packages/ssr/src/types.ts

```diff
@@ -2,13 +2,8 @@ import type { CookieSerializeOptions } from 'cookie';
 
 export type CookieOptions = Partial<CookieSerializeOptions>;
 export type CookieOptionsWithName = { name?: string } & CookieOptions;
-export type ServerCookieMethods = {
-	get: (key: string) => Promise<string | null | undefined> | string | null | undefined;
-	set: (key: string, value: string, options?: CookieOptions) => Promise<void> | void;
-	remove: (key: string, options?: CookieOptions) => Promise<void> | void;
-};
-export type BrowserCookieMethods = {
+export type CookieMethods = {
```

Comment:
Also realised the `options` parameter should not be optional. Both `createServerClient` and `createBrowserClient` pass in the default cookie options when they call these functions